docs/ovms_quickstart.md

```diff
@@ -0,0 +1,101 @@
+# OpenVINO&trade; Model Server Quickstart
+
+To quickly start using OpenVino&trade; Model Server please execute the following steps.
+
+### Deploy Model Server
+
+The Docker image with OpenVino&trade; Model Server is available on the DockerHub. Issue the
+following command to start using it:
+
+```bash
+docker pull openvino/ubuntu18_model_server
+```
+There are also other ways to obtain Model Server application:
+* Building a container image with a Dockerfile - more details can be found [here](docker_container.md)
+* Building the application locally from source - detailed description is located [here](host.md)  
+
+### Download a desired model
+
+OpenVino&trade; Model Server requires a trained model to be able to perform an inference. The model
+must be in IR format - a pair of files with .bin and .xml extensions. A model 
+can be downloaded from various sites in the Internet (for example from the [Open Model Zoo](https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/) ) or converted from other formats - like TensorFlow, ONNX, Caffe or MXNet using [Model Optimizer](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) which is a part of the OpenVINO Toolkit.
+
+Let's use the face detection model available under the following [link](https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/). 
+
+Then download both components of the model to the `model` directory (for example using curl):
+```
+curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/face-detection-retail-0004.xml -o model/face-detection-retail-0004.bin
+```
+
+### Start the Model Server container
+
+To start the Model Server container use the following command:
+
+```bash
+docker run -d -v <folder_with_downloaded_model>:/models/face-detection/1 -e LOG_LEVEL=DEBUG -p 9000:9000 openvino/ubuntu18_model_server \
+/ie-serving-py/start_server.sh ie_serving model --model_path /models/face-detection --model_name face-detection --port 9000  --shape auto
+```
+
+Model Server expects models in the predefined structure of folders - that is why the folder with downloaded models is mounted as `/models/face-detection/1`. In our case this expected structure is as follows:
+
+```bash
+models/
+└── face-detection
+    └── 1
+        ├── face-detection-0106.bin
+        └── face-detection-0106.xml
+
+``` 
+
+More about this structure and about how to deploy more than one model at the time can be found [here](./docker_container.md#preparing-the-models) and [here](./docker_container.md#starting-docker-container-with-a-configuration-file).
+
+### Download the example client script

```

Comment:
Please change to `### Download the Example Client Script`

Reply:
corrected